prepared.go

```diff
@@ -117,6 +140,17 @@ func (req *ExecutePreparedRequest) Body(res SchemaResolver, enc *msgpack.Encoder
 	return fillExecutePrepared(enc, *req.stmt, req.args)
 }
 
+// Context sets a passed context to the request.
+//
+// Pay attention that when using context with request objects,
+// the timeout option for Connection does not affect the lifetime
+// of the request. For those purposes use context.WithTimeout() as
+// the root context.
```

Comment:
We can add the same note to (or just to instead of each request) Connection.opts.Timeout description.

Reply:
I have checked, the go doc looks better so. Also added to opts.timeout description.